api/src/jvmMain/kotlin/com/buenaflor/kflogger/Klass.jvm.kt

```diff
@@ -0,0 +1,9 @@
+package com.buenaflor.kflogger
+
+import kotlin.reflect.KClass
+
+public actual typealias Klass<T> = Class<T>
+
+public actual fun <T : Any> KClass<T>.toKlass(): Klass<T> {
+  return this.java
+}
```

Comment:
`Klass` workaround to make `actual typealias` work with `Platform`

Reply:
Looks good, reflection is one of the rough edges for KMP